float32 gazebo_odom
float32 self_odom
float32 dist_odom
float32 pose11
float32 pose33
float32 pose32
float32 self_err
float32 dist_err
float32 distance_11_mean
float32 distance_33_mean
float32 distance_32_mean
float32 distance_11_median
float32 distance_33_median
float32 distance_32_median
float32 pos1
float32 pos2
float32 pos3